.github/workflows/build-and-test.yml

```
@@ -460,7 +460,7 @@ jobs:
 
   build-package:
     # Use 20.04.5 until https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/16450 is resolved
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

There's an issue to deal with this upgrade, currently not supported

```suggestion
    runs-on: ubuntu-20.04
```